app/views/mail/register.scala.html

```diff
@@ -1,5 +1,62 @@
-@( name: String, brainDBresult: Option[String], enableAutoVerify: Boolean ) Hello @name, thank you
-for signing up for an webKnossos account! @if(!enableAutoVerify) { Your account is going to get
-activated after an administrator has reviewed your details. } @if(brainDBresult.isDefined) {
-@brainDBresult.get } Enjoy using webKnossos Your webKnossos Team
+@(name: String, brainDBresult: Option[String], enableAutoVerify: Boolean ) 
 
+@emailBaseTemplate() {
+
+  <p>Hi @name,</p>
+
+  @if(!enableAutoVerify) { 
+    <p>
+      welcome to your new webKnossos account. You are almost ready to start exploring and to collaboratively
+      annotate large 3D datasets.
+    </p>
+    <p>Your account is currently under review. An administrator will activate you soon.</p>
+  } else {
+    <p>
+      welcome to your new webKnossos account. Get ready to start exploring and to collaboratively
+      annotate large 3D datasets.
+    </p>
+  }
+
+  @if(brainDBresult.isDefined) { 
+    <p>@brainDBresult.get</p>
+  } 
+
+  <h2 style="font-size: 1rem;">Get started</h2>
+  <ul style="font-size: 1rem;">
+    <li>
+      Use webKnossos for free with any of the public datasets on
+      <a href="https://webknossos.org">webknossos.org</a>. All webKnossos features can be used
+      with your new free account.
+    </li>
+    <li>
+      Upload your own dataset into your own space private workspace. If desired, you can
+      publish your dataset into our public gallery.
+    </li>
+    <li>
+      Invite colleagues from your lab or external collaborators to work with your data.
+      webKnossos works great for small teams and supports management of large annotator crowds.
+    </li>
+  </ul>
+
+  <h2 style="font-size: 1rem;">Get support</h2>
+  <p>
+    To learn more about how to get the most out of webKnossos,
+    <a href="https://docs.webknossos.org">check out the documentation</a> or
+    <a href="https://www.youtube.com/watch?v=tNwDvo5MnDc">watch this introductory video</a>.
+  </p>
+  <p>
+    <a href="https://docs.webknossos.org"><img src="https://static.webknossos.org/mails/screen-docs.png"
+        alt="screenshot of documentation" width="200" height="137" /></a>
+    <a href="https://www.youtube.com/watch?v=tNwDvo5MnDc"><img src="https://static.webknossos.org/mails/screen-video.png"
+        alt="screenshot of intro video" width="200" height="137" /></a>
+  </p>
+
+  <p>
+    Please don’t hesitate to get in touch with me if you have any questions about using
+    webKnossos.
+  </p>
+
+  <p>I am looking forward to hearing from you.</p>
+
+  <p>Best regards,<br />Norman</p>
```

Comment:
Again this reads like it's directed to webknossos.org users, but isn't this the general email for webKnossos registrations on any instance?
Should this be `<p>With best regards,<br />Your webKnossos Team</p>` as well?

Reply:
I intentionally left this part. If every some decides to install wK somewhere, they might want to get in touch with us. And for current lab user, this still has a nice personal touch?